tests/unit/commands/test_napps_api.py

```diff
@@ -180,6 +182,35 @@ def test_install_napp(self, mock_napps_manager):
 
         mgr.remote_install.assert_called()
 
+    @patch('kytos.utils.napps.NAppsManager')
+    def test_install_napp_error_404(self, mgr):
+        """Test install_napp method."""
+        mgr = MagicMock()
+
+        url = 'www.napps.kytos.io'
```

Comment:
There's no 'www' here.

```suggestion
        url = 'napps.kytos.io'
```